app/views/layouts/_topbar.html.haml

```diff
@@ -13,8 +13,8 @@
     / Right Nav Section
     %ul.right
       %li.active
-        %a{:href => "#"} Right Button Active
         - if user_signed_in?
+          %a{href: "#", id: "#{current_user.id}", class: "username"}= "Signed in as #{current_user.username}"
```

Comment:
`= "..."` should be rewritten as `...`<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [109/80]<br>Avoid defining `class` in attributes hash for static class names